tests/integration_tests/conftest.py

```diff
@@ -107,26 +107,32 @@ def data_db_connection() -> Generator[Connection, None, None]:
 
 
 @pytest.fixture(scope="function")
-def data_db(data_db_connection, monkeypatch):
-
-    transaction = data_db_connection.begin()
-    print(f"This test is being performed with transaction {transaction}")
-
-    SessionLocal = sessionmaker(
-        autocommit=False, autoflush=False, bind=data_db_connection
-    )
-    session = SessionLocal()
-
-    def get_test_db():
-        return session
-
-    monkeypatch.setattr(db_session, "get_db", get_test_db)
-
-    yield session
-
-    session.close()
-    print(f"This test is finished and being rolledback with transaction {transaction}")
-    transaction.rollback()
+def data_db(slow_db):
```

Comment:
Old function commented out below - could add a comment - but I think we both know the history of this ... pain :crossed_swords:

Reply:
Do we want to keep it commented out?

Reply:
I think so for now - it keeps out thoughts alive - I still would like to do more work on this when I can... but if you insist I;'ll remove